x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/common/notifications/Level.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.core.common.notifications;
+
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.io.stream.Writeable;
+
+import java.io.IOException;
+import java.util.Locale;
+
+public enum Level implements Writeable {
+    INFO, ACTIVITY, WARNING, ERROR;
```

Comment:
Where would we use `activity` instead of `info`?

Reply:
It's true that we don't use this at all in the current product.  In the old product where we did use it it is interesting to see who added it:

```
f52b8ead288 (Dimitrios Athanasiou 2016-05-05 13:36:31 +0100 101)                 auditor.activity(buildUsageMessage(activeJobs));
```

In other words, the usage information similar to what the X-Pack usage endpoint returns and is now stored by monitoring used to be stored in the notifications index with type `activity`.

But assuming I'm correct that we don't use it any more (please double check @benwtrent) now would be a good time to remove it.

Reply:
One more thing about removing activity - usually this would be a BWC breaking change.  But in this case I don't think we ever serialise `Level` or `AuditMessage` on the wire, so we can do it.  This leads to the question, should we even support serialising `Level` or `AuditMessage` on the wire?  If you remove wire serialisation from `Level` and `AuditMessage` and their tests does the code still compile?

Reply:
@droberts195 @dimitris-athanasiou I tested on ML by removing serialization and `ACTIVITY` and there are no calls to serialization for `AuditMessage` and therefore no calls to `Level` serialization.